plugins/notifications/api-report.md

```diff
@@ -182,7 +183,10 @@ export function useTitleCounter(): {
 };
 
 // @public (undocumented)
-export function useWebNotifications(enabled: boolean): {
+export function useWebNotifications(
```

Comment:
Hmm, how come this is exported in the first place, should we keep it internal? A think like this should otherwise live in `notifications-react`. If we want to keep it exported, let's make it an options object instead: `useWebNotifications(options: {enabled?: boolean, markAsReadOnLinkOpen?: boolean})`

Reply:
Yeah, it probably should be internal, I will change!